tests/deit/console/option/parser/ArgvParserTest.php

```diff
@@ -11,14 +11,14 @@ class ArgvParserTest extends \PHPUnit_Framework_TestCase {
 
 	public function test() {
 
-		$console    = new GenericConsole();
+		$console    = new Event();
 		$argv       = array(
-			'my-cmd.php',
-			'-abc',
-			'-d=v',
-			'-test',
-			'--opt=test',
-		);
+							'my-cmd.php',
```

Comment:
It's not a job for tools but for the editor.  The editor basically has to detect what style of tabs and usage has been used in a particular source file and change the editor settings to comply.

The github view of this change looks like I deleted those lines out of the test, but if you click on "View Full Changes" in the box with the change, you'll see there are lines added.  Basically just whitespace change.

Which brings up a different problem.  I think I (or my editor) put tabs in this file because they were the leading whitespace.  But that doesn't work for lining things up with previous lines.  So it looks great on my screen with tabstop at 4 spaces, but bad here on Github with 8 space tabstop (and not right with any tabstop other than 4 spaces).

So what are the rules?

1) Leading tab characters if being used for indent
2) But leading space characters if being used for alignment that's more than indenting (like here), but the same number of initial tab characters must be used as the line above (so we'll always have mixed leading tabs and spaces if lining up with a previously indented line!).
3) Spaces everywhere else.

Something like that?